mlflow/ml-package-versions.yml

```diff
@@ -203,9 +203,17 @@ catboost:
   package_info:
     pip_release: "catboost"
     install_dev: |
-      # TODO: Remove the commit once https://github.com/catboost/catboost/issues/2242 is fixed
-      pip install \
-        git+https://github.com/catboost/catboost.git@f8921d839eccf11e533a46347dde5b5b25b17190#subdirectory=catboost/python-package
+      # Install CMake
+      CMAKE_VERSION=3.24.0
+      wget https://github.com/Kitware/CMake/releases/download/v$CMAKE_VERSION/cmake-$CMAKE_VERSION-linux-x86_64.sh
+      chmod +x cmake-$CMAKE_VERSION-linux-x86_64.sh
+      sudo ./cmake-$CMAKE_VERSION-linux-x86_64.sh --skip-license --prefix=/usr/local
+      # Install Ninja
+      sudo apt-get update
+      sudo apt-get install -y ninja-build
+
+      # Install catboost from source
+      pip install git+https://github.com/catboost/catboost.git#subdirectory=catboost/python-package 'numpy<2'
```

Comment:
Let's remove `install_dev`. This will fail again in the future.

Reply:
lol ok